Dockerfile-ubuntu.template

```diff
@@ -23,12 +23,15 @@ RUN apt-get update && \
 	apt-get clean && \
 	rm -rf /var/lib/apt/lists/* 
 
+RUN useradd -ms /bin/bash idsvr
```

Comment:
Does this user really need bash and a home directory? If not, pass `--system` and drop the shell.

Reply:
yes it does